src/com/codecool/Trapper.java

```diff
@@ -0,0 +1,37 @@
+package com.codecool;
+
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+
+public class Trapper {
+    private static final int MAX_PAGES_TO_SEARCH = 1000000000;
+    private Set<String> pagesVisited = new HashSet<String>();
+    private LinkedList<String> pagesToVisit = new LinkedList<>();
+    int visitedPages = 0;
+
+    private void addUrl(String url){
+
+    }
+
+    private String nextUrl(){
+        return "";
+    }
+
+    public void search(String url, String searchedWord, String searchedTitle){
+        this.addUrl(url);
+        long start = System.currentTimeMillis();
+
+        while(this.pagesVisited.size() < MAX_PAGES_TO_SEARCH)
+        {
+            String currentUrl;
+            HTTPHandler handler = new HTTPHandler();
+
+            // TODO: use HTTP class to crawl over sites
+
+        }
+        System.out.println(String.format("**Done** Visited %s web page(s)", visitedPages));
+        System.out.println("Searching took:  " + (System.currentTimeMillis() - start) + " ms");
```

Comment:
Seconds might be more user friendly?
```suggestion
        float seconds = (System.currentTimeMillis() - start) / 1000.0;
        System.out.println("Searching took:  " + seconds + " s");
```